_health-care/_js/components/financial-assessment/AnnualIncomeSection.jsx

```diff
@@ -148,4 +150,21 @@ class AnnualIncomeSection extends React.Component {
   }
 }
 
-export default AnnualIncomeSection;
+function mapStateToProps(state) {
+  return {
+    data: state.annualIncome,
+    receivesVaPension: state.vaInformation.receivesVaPension
+  };
+}
+
+function mapDispatchToProps(dispatch) {
+  return {
+    onStateChange: (field, update) => {
+      dispatch(updateField(['annualIncome', field], update));
+    }
+  };
+}
+
+// TODO(awong): Remove the pure: false once we start using ImmutableJS.
+export default connect(mapStateToProps, mapDispatchToProps, undefined, { pure: false })(AnnualIncomeSection);
```

Comment:
Yes, this creates a connect component. Specifically, it names itself `Connect(AnnualIncomeSection)`.

As for the undefined...my understanding of the API doc is passing in "undefined" actually yields the default behavior.  I only put that in there cause I needed to specify `{pure : false}`.  Looking at it more though, I wonder if there's a clever way of using that to avoid needing VETERAN_UPDATE_FIELD to know the field path... hmmm... something to think about for later.